src/pages/Explore/explore.css

```diff
@@ -0,0 +1,89 @@
+.explore-main .container {
+    gap: var(--space-3);
+    padding: var(--space-3) var(--space-1)
+}
+
+.category-list {
+   gap: var(--space-1);
+}
+
+.category-list .badge {
+    position: relative;
+}
+
+.videos-list-container {
+    display: grid;
+    width: 100%;
+    grid-template-columns: 1fr;
+    gap: var(--space-2);
+}
+
+.App.dark .video-card:hover {
+    background-color: var(--dark-theme-navbar-bg-color);
+}
+
+.App.light .video-card:hover {
+    background-color: var(--light-theme-navbar-bg-color);
+}
+
+.video-card {
+    border-radius: var(--br-2);
+    -webkit-border-radius: var(--br-2);
+    -moz-border-radius: var(--br-2);
+    -ms-border-radius: var(--br-2);
+    -o-border-radius: var(--br-2);
+    background-color: inherit;
+    width: 100%;
+    max-width: 100%;
+    transition: all 0.3s ease-in-out;
+    -webkit-transition: all 0.3s ease-in-out;
+    -moz-transition: all 0.3s ease-in-out;
+    -ms-transition: all 0.3s ease-in-out;
+    -o-transition: all 0.3s ease-in-out;
+}
+
+.card-header, .video-img {
+    border-radius: var(--br-2);
+    -webkit-border-radius: var(--br-2);
+    -moz-border-radius: var(--br-2);
+    -ms-border-radius: var(--br-2);
+    -o-border-radius: var(--br-2);
+}
+
+.video-card .card-body {
+    background-color: inherit;
+    color: inherit;
+    width: 100%;
+    flex: 1.5;
+    border-radius: var(--br-2);
+    -webkit-border-radius: var(--br-2);
+    -moz-border-radius: var(--br-2);
+    -ms-border-radius: var(--br-2);
+    -o-border-radius: var(--br-2);
+}
```

Comment:
Suggestion:  I saw that you are using the same border-radius at most of the places. You can directly create a class for it and add that class instead. 

I think it will reduce a lot of code as every time you are defining it for all the browser

Reply:
This is a great idea. Thank you. I'll do this 👍🏽

Reply:
I have made the changes.